services/src/main/java/org/exoplatform/webconferencing/WebConferencingService.java

```diff
@@ -2109,7 +2109,7 @@ public void uploadFile(UploadFileInfo uploadInfo, HttpServletRequest request) th
         } else {
           saveFile(rootNode, resource, uploadingUser, null);
         }
-        broacastCallEvent(EVENT_CALL_RECORDED, call, uploadingUser, resource.getUploadedSize(),resource.getFileName(), uploadInfo, STATUS_OK);
+        broacastCallEvent(EVENT_CALL_RECORDED, call, owner, resource.getUploadedSize(),resource.getFileName(), uploadInfo, STATUS_OK);
```

Comment:
By changing this, you change the analytics which are create when a user upload a recording.
In analytics, we want the uploading user, not the owner
So this solution seems ko for me

Reply:
> By changing this, you change the analytics which are create when a user upload a recording. In analytics, we want the uploading user, not the owner So this solution seems ko for me

it causes some problems just in space calls in module analytic and it is not used in code (in the Call Recording NotificationListener) so I reset this parameter to the old variable uploading User then I test it now and everything is ok